Mod1/Labs/Week1/IntroToLabs.md

```diff
@@ -26,14 +26,14 @@ As you review and discuss, make note of any lingering questions you may have, or
 
 ### Preparation
 
-Tomorrow, we will be discussing one of the most important fundamentals of programming, Datatypes and Variables.  To prepare for that lesson, watch the video linked below, and discuss the check for understanding question with your group.
+Tomorrow, we will be discussing one of the most important fundamentals of programming, Datatypes and Variables. To prepare for that lesson, watch the video linked below, and discuss the check for understanding question with your group.
 
 [![video image link](/Mod1/Images/Week1/Variables_ProgrammingForBeginners.png)](https://www.youtube.com/watch?v=ghCbURMWBD8)
 
 #### Check for Understanding
 * Which of the following real life items is most similar to a **variable**:
     - a book
-    - the label 'BOOKS' on a box of books
+    - a box labeled 'Nonfiction' that contains books
```

Comment:
I like the label 'Nonfiction', but I want students to identify that it is the **label** that is most similar to a variable (not the actual box).